--- a/src/MainWindow.vala	2019-09-28 07:13:39.000000000 +0200
+++ b/src/MainWindow.vala	2019-10-11 00:25:54.931542031 +0200
@@ -69,6 +69,10 @@
             insert_action_group ("win", actions);
 
             var settings = AppSettings.get_default ();
+            
+            Gtk.Settings.get_default().set_property("gtk-theme-name", "elementary");
+            Gtk.Settings.get_default().set_property("gtk-icon-theme-name", "elementary");
+            
             int x = settings.window_x;
             int y = settings.window_y;
             int h = settings.window_height;
